app/src/androidTest/java/com/github/h3lp3rs/h3lp/forum/ForumAnswersActivityTest.kt

```diff
@@ -37,8 +37,8 @@ const val ANSWER_TEST = "answer"
 
 @RunWith(AndroidJUnit4::class)
 class ForumAnswersActivityTest {
-    private lateinit var forum : Forum
-    private lateinit var proUsersDb : Database
+    private lateinit var forum: Forum
```

Comment:
I can't comment on it but it'd be better to put CATEGORY_TEST_STRING = TRAUMATOLOGY.name for the code to be modular in case we switch categories / category names

Reply:
Tried it and does not work since the variable is outside the class :(